releases/development/master/extra/install-chef-suse.sh

```diff
@@ -634,11 +634,15 @@ if [ -z "$CROWBAR_FROM_GIT" ]; then
     fi
 fi
 
-check_or_create_ptf_repository 11.3 Cloud-PTF
-check_or_create_ptf_repository 12.0 Cloud-PTF
-# Currently we only sign the Cloud-PTF repository
-sign_repositories 11.3 Cloud-PTF
-sign_repositories 12.0 Cloud-PTF
+# TODO: do we need a Storage-PTF repo?
```

Comment:
I'd say yes. Or we could actually just name the repo `PTF` (a change that'd be fine in Cloud 6) and have everyone share it :-)